modules/ob-build-controller-metrics.adoc

```diff
@@ -0,0 +1,35 @@
+// This module is included in the following assembly:
+//
+// builds/build-controller-observability.adoc
+
+:_content-type: REFERENCE
+[id="ob-build-controller-metrics_{context}"]
+= Build controller metrics
+
+You can check the following build controller metrics for monitoring purposes:
+
+
+.Build controller metrics
+[options="header", cols="30%,10%,30%,30%,10%"]
+|===
+
+| Name | Type | Description | Labels | Status
+
+| `build_builds_registered_total` | Counter | The number of total registered builds. | buildstrategy=<build_buildstrategy_name> namespace=<buildrun_namespace>
+build=<build_name>  | experimental
+
+| `build_buildruns_completed_total` | Counter | The number of total completed build runs. | buildstrategy=<build_buildstrategy_name> namespace=<buildrun_namespace> build=<build_name> buildrun=<buildrun_name> | experimental
+
+| `build_buildrun_establish_duration_seconds` | Histogram | The build run establish duration in seconds. | buildstrategy=<build_buildstrategy_name> 
+namespace=<buildrun_namespace> build=<build_name> buildrun=<buildrun_name> | experimental
+
+| `build_buildrun_completion_duration_seconds` | Histogram | The build run completion duration in seconds. | buildstrategy=<build_buildstrategy_name> namespace=<buildrun_namespace> build=<build_name> buildrun=<buildrun_name>  | experimental
+
+| `build_buildrun_rampup_duration_seconds` | Histogram | The build run ramp-up duration in seconds. | buildstrategy=<build_buildstrategy_name> 
+namespace=<buildrun_namespace> build=<build_name> buildrun=<buildrun_name> | experimental
+
+| `build_buildrun_taskrun_rampup_duration_seconds` | Histogram | The build run ramp-up duration for a task run in seconds. | buildstrategy=<build_buildstrategy_name> namespace=<buildrun_namespace> build=<build_name> buildrun=<buildrun_name>  | experimental
+
+| `build_buildrun_taskrun_pod_rampup_duration_seconds` | Histogram | The build run ramp-up duration for a task run pod in seconds. | buildstrategy=<build_buildstrategy_name> namespace=<buildrun_namespace> build=<build_name> buildrun=<buildrun_name>  | experimental
+
+|===
```

Comment:
The **Labels** column in this was a little hard to quickly parse with all of them being verbatim text. It might be helpful (if the following still is technically accurate) to markup the individual labels and then add an `a` (for ["AsciiDoc" mode](https://docs.asciidoctor.org/asciidoc/latest/tables/format-cell-content/#a-operator)) before the `|` cells for those entries so that you can use unordered lists to show some separation. For example, something like:
```suggestion
.Build controller metrics
[options="header", cols="30%,10%,30%,30%,10%"]
|===

| Name | Type | Description | Labels | Status

| `build_builds_registered_total`
| Counter
| The number of total registered builds.
a|* `buildstrategy=<build_buildstrategy_name>`
* `namespace=<buildrun_namespace>`
* `build=<build_name>`
| experimental

| `build_buildruns_completed_total`
| Counter
| The number of total completed build runs.
a|* `buildstrategy=<build_buildstrategy_name>`
* `namespace=<buildrun_namespace>`
* `build=<build_name>`
* `buildrun=<buildrun_name>`
| experimental

| `build_buildrun_establish_duration_seconds`
| Histogram
| The build run establish duration in seconds.
a|* `buildstrategy=<build_buildstrategy_name>`
* `namespace=<buildrun_namespace>`
* `build=<build_name>`
* `buildrun=<buildrun_name>`
| experimental

| `build_buildrun_completion_duration_seconds`
| Histogram
| The build run completion duration in seconds.
a|* `buildstrategy=<build_buildstrategy_name>`
* `namespace=<buildrun_namespace>`
* `build=<build_name>`
* `buildrun=<buildrun_name>`
| experimental

| `build_buildrun_rampup_duration_seconds`
| Histogram
| The build run ramp-up duration in seconds.
a|* `buildstrategy=<build_buildstrategy_name>`
* `namespace=<buildrun_namespace>`
* `build=<build_name>`
* `buildrun=<buildrun_name>`
| experimental

| `build_buildrun_taskrun_rampup_duration_seconds`
| Histogram
| The build run ramp-up duration for a task run in seconds.
a|* `buildstrategy=<build_buildstrategy_name>`
* `namespace=<buildrun_namespace>`
* `build=<build_name>`
* `buildrun=<buildrun_name>`
| experimental

| `build_buildrun_taskrun_pod_rampup_duration_seconds`
| Histogram
| The build run ramp-up duration for a task run pod in seconds.
a|* `buildstrategy=<build_buildstrategy_name>`
* `namespace=<buildrun_namespace>`
* `build=<build_name>`
* `buildrun=<buildrun_name>`
| experimental

|===
```

This comes out looking like:
<img width="546" alt="image" src="https://github.com/openshift/openshift-docs/assets/3442316/d5fea387-f216-4a40-806b-3779e939e461">